source/client/components/CVLanguageManager.ts

```diff
@@ -46,7 +47,11 @@ export default class CVLanguageManager extends Component
 
     protected static readonly ins = {
         enabled: types.Boolean("Language.Enabled", false),
-        language: types.Enum("Interface.Language", ELanguageType, ELanguageType[DEFAULT_LANGUAGE]),
+        language: types.Enum("Interface.Language", ELanguageType, {
```

Comment:
Can you be more specific about the issue this is addressing? Property views in the UI should already be showing the full text except for the Explorer button that opens the language menu.

Reply:
Never mind, I see this supports the other task changes.

Reply:
Those properties were using a mirror of this property to provide the full text. This required a lot `synchLanguage()` that were prone to misuse. Seeing that removing all those properties ([49cf2a4](https://github.com/Smithsonian/dpo-voyager/pull/280/commits/49cf2a4b4ec0903430a7fef353e14d3f55381754)..[a0dd844](https://github.com/Smithsonian/dpo-voyager/pull/280/commits/a0dd84469327779ddea8f73f367f709474036010)) was really simple I thought it more elegant to always rely on CVLanguageManager's property. However the selectors then displayed to language's codeString ("FR", "EN"), which was a regression that this fixes preventively.